diff --git a/node_modules/svelvet/Containers/GraphView/index.svelte b/node_modules/svelvet/Containers/GraphView/index.svelte
index 328a419..2433bad 100644
--- a/node_modules/svelvet/Containers/GraphView/index.svelte
+++ b/node_modules/svelvet/Containers/GraphView/index.svelte
@@ -22,7 +22,7 @@ export let derivedEdges;
 export let key;
 // here we lookup the store using the unique key
 const svelvetStore = findOrCreateStore(key);
-const { nodeSelected, backgroundStore, widthStore, heightStore, d3Scale } = svelvetStore;
+const { nodeSelected, backgroundStore, movementStore, widthStore, heightStore, d3Scale } = svelvetStore;
 // declaring the grid and dot size for d3's transformations and zoom
 const gridSize = 15;
 const dotSize = 10;
@@ -38,6 +38,8 @@ let d3Zoom = d3
     .on('zoom', handleZoom);
 // function to handle zoom events - arguments: d3ZoomEvent
 function handleZoom(e) {
+  if (!$movementStore) return
+  
     //add a store that contains the current value of the d3-zoom's scale to be used in onMouseMove function
     d3Scale.set(e.transform.k);
     // should not run d3.select below if backgroundStore is false
diff --git a/node_modules/svelvet/Containers/Svelvet/index.svelte b/node_modules/svelvet/Containers/Svelvet/index.svelte
index a6e029e..aa01459 100644
--- a/node_modules/svelvet/Containers/Svelvet/index.svelte
+++ b/node_modules/svelvet/Containers/Svelvet/index.svelte
@@ -8,6 +8,7 @@ export let edges;
 export let width = 600;
 export let height = 600;
 export let background = false;
+export let movement = true;
 // generates a unique string for each svelvet component's unique store instance
 const key = (Math.random() + 1).toString(36).substring(7);
 // creates a store that uses the unique sting as the key to create and look up the corresponding store
@@ -22,6 +23,7 @@ onMount(() => {
     svelvetStore.widthStore.set(width);
     svelvetStore.heightStore.set(height);
     svelvetStore.backgroundStore.set(background);
+    svelvetStore.movementStore.set(movement);
 });
 // enables data reactivity
 afterUpdate(() => {
@@ -30,6 +32,7 @@ afterUpdate(() => {
     svelvetStore.widthStore.set(width);
     svelvetStore.heightStore.set(height);
     svelvetStore.backgroundStore.set(background);
+    svelvetStore.movementStore.set(movement);
 });
 </script>
 
diff --git a/node_modules/svelvet/Containers/Svelvet/index.svelte.d.ts b/node_modules/svelvet/Containers/Svelvet/index.svelte.d.ts
index 5e01ad9..6e8e8f0 100644
--- a/node_modules/svelvet/Containers/Svelvet/index.svelte.d.ts
+++ b/node_modules/svelvet/Containers/Svelvet/index.svelte.d.ts
@@ -7,6 +7,7 @@ declare const __propDef: {
         width?: number | undefined;
         height?: number | undefined;
         background?: boolean | undefined;
+        movement?: boolean | undefined;
     };
     events: {
         [evt: string]: CustomEvent<any>;
diff --git a/node_modules/svelvet/Nodes/index.svelte b/node_modules/svelvet/Nodes/index.svelte
index 0bc4156..78c8b9d 100644
--- a/node_modules/svelvet/Nodes/index.svelte
+++ b/node_modules/svelvet/Nodes/index.svelte
@@ -1,7 +1,16 @@
 <script>import { findOrCreateStore } from '../stores/store';
 export let node;
 export let key;
-const { onMouseMove, onNodeClick, onTouchMove, nodeSelected, nodeIdSelected } = findOrCreateStore(key);
+const {
+    onMouseMove,
+    onNodeClick,
+    onTouchMove,
+    nodeSelected,
+    nodeIdSelected,
+    movementStore
+    } = findOrCreateStore(key);
+
+  $: shouldMove = moving && $movementStore;
 // $nodeSelected is a store boolean that lets GraphView component know if ANY node is selected
 // moving local boolean specific to node selected, to change position of individual node once selected
 let moving = false;
@@ -11,7 +20,7 @@ let moved = false;
 <svelte:window
   on:mousemove={(e) => {
     e.preventDefault();
-    if (moving) {
+    if (shouldMove) {
       onMouseMove(e, node.id);
       moved = true;
     }
@@ -20,7 +29,7 @@ let moved = false;
 
 <div
   on:touchmove={(e) => {
-    if (moving) {
+    if (shouldMove) {
       onTouchMove(e, node.id);
     }
   }}
diff --git a/node_modules/svelvet/stores/store.d.ts b/node_modules/svelvet/stores/store.d.ts
index bd76794..0bcbf50 100644
--- a/node_modules/svelvet/stores/store.d.ts
+++ b/node_modules/svelvet/stores/store.d.ts
@@ -6,6 +6,7 @@ interface CoreSvelvetStore {
     widthStore: Writable<number>;
     heightStore: Writable<number>;
     backgroundStore: Writable<boolean>;
+    movementStore: Writable<boolean>;
     nodeIdSelected: Writable<number>;
     nodeSelected: Writable<boolean>;
     d3Scale: Writable<number>;
diff --git a/node_modules/svelvet/stores/store.js b/node_modules/svelvet/stores/store.js
index 54c2ca7..e3f9ec4 100644
--- a/node_modules/svelvet/stores/store.js
+++ b/node_modules/svelvet/stores/store.js
@@ -14,6 +14,7 @@ export function findOrCreateStore(key) {
         widthStore: writable(600),
         heightStore: writable(600),
         backgroundStore: writable(false),
+        movementStore: writable(true),
         nodeSelected: writable(false),
         nodeIdSelected: writable(-1),
         d3Scale: writable(1)
